### PR TITLE
Thanks contribubtor and feedback form link

### PR DESCRIPTION
**Pull request by Mariatta**

### CODE REVIEW

File: email_campaigns/thanks-contributor-email.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+excellent, and for contributing to this amazing community. We're inspired to see you
+all learning and contributing to open source during the sprint, and we
+hope you've been inspired to continue contributing. The Python community is better
+because of you 💜
+
+This is our first time organizing the mentored sprint, and numerous people
+have asked when will we do it again! Before we can get to that, we'd like to hear
+about your experience as a new contributor at the sprint.
+
+We are including an anonymous survey to hear your thoughts on what we can do even better next time.
```
Comment (Nikoleta-v3) [inline email_campaigns/thanks-contributor-email.md]: Same comments as the other `PR`

File: email_campaigns/thanks-contributor-email.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+have asked when will we do it again! Before we can get to that, we'd like to hear
+about your experience as a new contributor at the sprint.
+
+We are including an anonymous survey to hear your thoughts on what we can do even better next time. 
+
+Give Feedback: http://bit.ly/contributors-feedback-mentoredsprints
+
+# Thanks to mentors
+
+We're thankful to our mentors for volunteering their time and sharing their knowledge
```
Comment (Nikoleta-v3) [inline email_campaigns/thanks-contributor-email.md:32]: Can we include a link where all the mentors are listed?

File: email_campaigns/thanks-contributor-email.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+and passion with all of us. Please take a moment to give them a shout out and celebrate
+success using the [#mentoredsprint #pycon2019](https://twitter.com/search?q=%23mentoredsprint%20%23pycon2019&src=typd) hashtag on Twitter.
+
+# Thanks to y'all
```
Comment (nnja) [inline email_campaigns/thanks-contributor-email.md]: Same comment here https://github.com/pycon-mentored-sprints/write_ups/pull/8#discussion_r282603848

File: email_campaigns/thanks-contributor-email.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+The mentored sprint for diverse beginners was a great event only because you showed
+up and were the best version of yourself. Thanks for contributing to the community!
+
+# Thanks to PyCon staff and The PSF
+
+We want to extend another big thank you to PyCon Hatchery committee, PyCon staff members
+and The PSF for their support in making the Mentored Sprint for Diverse Beginners a success.
+
+From Tania, Nina, Nikoleta & Mariatta
```
Comment (nnja) [inline email_campaigns/thanks-contributor-email.md:54]: Should we include a way to keep this community active before the next event?

Mailing list? Newsletter for news and updates? Slack channel somewhere (not a new slack)?

**Review by nnja — CHANGES_REQUESTED** [review]
minor nit and question

File: email_campaigns/thanks-contributor-email.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+about your experience as a new contributor at the sprint.
+
+We are including an anonymous survey to hear your thoughts on what we can do even
+better next time. 
+
+**Give Your Feedback**: http://bit.ly/contributors-feedback-mentoredsprints
+
+# Reflections
+
+We're aware of several issues occurred during the mentored sprint, such as late
```
Comment (nnja) [inline email_campaigns/thanks-contributor-email.md]:
```suggestion
We're aware of several issues that occurred during the mentored sprint, such as a late
```

**Comment by nnja** [comment]
👍 from me pending newsletter details from @trallard

File: email_campaigns/thanks-contributor-email.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+# Thanks to PyCon staff and The PSF, and PyCon volunteers
+
+We want to extend another big thank you to PyCon Hatchery committee, PyCon staff members, Git
+Helpdesk volunteers, and The PSF for their support in making the Mentored Sprint
+for Diverse Beginners a success.
+
+# Stay in touch
+
+To get updated about future mentored sprint for diverse beginners events, you
```
Comment (trallard) [inline email_campaigns/thanks-contributor-email.md]: To get updates about future mentored sprint for diverse beginners events, you

**Review by trallard — COMMENTED** [review]
All looks good to me

**Comment by Mariatta** [comment]
Email was sent June 1, 2019

**Pull request merged**